README.md

```diff
@@ -1,54 +1,239 @@
-# Taipy
-[![tests](https://github.com/Avaiga/taipy/actions/workflows/tests.yml/badge.svg)](https://github.com/Avaiga/taipy/actions/workflows/tests.yml)
-[![Python](https://img.shields.io/pypi/pyversions/taipy)](https://pypi.org/project/taipy)
-[![PyPI](https://img.shields.io/pypi/v/taipy.svg?label=pip&logo=PyPI&logoColor=white)](https://pypi.org/project/taipy)
+<br>
+<br>
 
-## License
-Copyright 2023 Avaiga Private Limited
+<img align="left" src="readme_img/readme_logo.png" alt="Taipy Logo" width="150" height="150" ></img>
+<br>
+#  Welcome to Taipy 
+<p align="left">
+    <a href="https://pypi.python.org/pypi/taipy/" alt="Taipy version">
+        <img alt="PyPI" src="https://img.shields.io/pypi/v/taipy.svg?label=pip&logo=PyPI&color=ff462b&labelColor=283282"></a>
+    <a href="https://pypi.org/project/taipy" alt="Python version">
+        <img alt="PyPI" src="https://img.shields.io/pypi/pyversions/taipy?color=ff462b&labelColor=283282"></a>
+    <a href="https://www.youtube.com/@taipy8009" alt="YouTube">
+        <img src="https://img.shields.io/badge/youtube-click_to_watch_videos-red.svg?color=ff462b&labelColor=283282&logo=youtube" /></a>
+     <a href="https://twitter.com/Taipy_io" alt="Twitter">
+        <img src="https://img.shields.io/badge/twitter-click_for_tweets-red.svg?color=ff462b&labelColor=283282&logo=twitter" /></a>
+    <a href="https://github.com/Avaiga/taipy/actions/workflows/tests.yml" alt="tests">
+        <img alt="PyPI" src="https://github.com/Avaiga/taipy/actions/workflows/tests.yml/badge.svg"></a>
 
-Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
-the License. You may obtain a copy of the License at
-[http://www.apache.org/licenses/LICENSE-2.0](https://www.apache.org/licenses/LICENSE-2.0.txt)
 
-Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
-an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
-specific language governing permissions and limitations under the License.
+<br>
+
+###  <div align="left">Turns Data and AI algorithms into full web apps in no time. 
```

Comment:
- I really don't like 'web' not being capitalized
- I really don't like 'apps', since the word refers more to games than to industrial applications.

Reply:
I agree with app being more for mobile apps. Web application sounds more industrial.

Reply:
Well I'll try to find a reasonable source that we can rely on. 
We will propagate the decision where it's needed.

I suspect we want a Glossary of some sort, across the project and the company.